build_tools/testing/update_tflite_models.py

```diff
@@ -0,0 +1,40 @@
+from absl import app
+from absl import flags
+from google.cloud import storage
+from google_auth_oauthlib import flow
+
+import tempfile
+import urllib
+
+FLAGS = flags.FLAGS
+
+flatbuffers = dict({
+    "mobilenet_v1.tflite":
+        "https://tfhub.dev/tensorflow/lite-model/mobilenet_v1_1.0_160/1/default/1?lite-format=tflite",
+    "posenet_i8.tflite":
+        "https://tfhub.dev/google/lite-model/movenet/singlepose/lightning/tflite/int8/4?lite-format=tflite",
+})
+
+BUCKET_NAME = "iree-model-artifacts"
+FOLDER_NAME = "tflite-integration-tests"
+
+
+def upload_model(source, destination, tmpfile):
+  """Uploads a file to the bucket."""
+  urllib.request.urlretrieve(source, tmpfile)
+
+  storage_client = storage.Client()
+  bucket = storage_client.get_bucket(BUCKET_NAME)
+  blob = bucket.blob("/".join([FOLDER_NAME, destination]))
+  blob.upload_from_filename(tmpfile)
+
+
+def main(argv):
+  tf = tempfile.NamedTemporaryFile()
+
+  for dst, src in flatbuffers.items():
+    upload_model(src, dst, tf.name)
+
+
+if __name__ == '__main__':
```

Comment:
The directory structure is fuzzy, but how do you feel about this sitting in [`scripts/`](https://github.com/google/iree/tree/main/scripts) instead of [`build_tools/testing/`](https://github.com/google/iree/tree/main/build_tools/testing)? (I don't have a strong preference, it partly depends on who is expected to be running this and how frequently it should be run)

Reply:
I thought about putting it in scripts, I just felt like it should be in something testing specific. I'll move the file there.

Reply:
As for frequency, it should only serve as documentation of the model sources / inputs. Reruns should only occur when new files are added. I wanted to guarantee we know where the original testfiles came from for our mirror (along with simplifying uploading new test files).